_docs/style/date-time.md

```diff
@@ -2,65 +2,104 @@
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Date format"] [class:"image bg-light border"] %}
 | Format | Examples |
 | --- | --- |
 | {month (mixed case 3 letter abbr)} {numerical day}, {year (4 digit)} | Nov 28, 2016<br>Apr 2, 2016 |
 | Use an en dash (&ndash) to separate the dates in date ranges. | Nov 28, 2016–Nov 30, 2016 |
 {: class="ui very basic collapsing celled table"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<hr>
 
-# Time
+### Time
 
-## Best Practices
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Best Practices
 
 -   Timezone control flow: Always convert to local time based user preference.
     If unavailable, use assumed time based on browser. Only use UTC when no
     timezone preference can be determined or standardization across timezones
     is necessary.
-
 -   Always convert to local time based user preference. If unavailable, use
     assumed time based on browser. Only use UTC when no timezone preference can
     be determined or standardization across timezones is necessary.
-
 -   Always offer users the ability to convert to UTC
-
 -   Add UTC offset to timezone only when that specificity is important (for
     example, "CST (UTC-6)").
 
--   Add seconds or milliseconds to the time only when that level specificity is
-    important
+{% endcolumn %}
 
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Time format"] [class:"image bg-light border"] %}
 | Format | Examples |
 | --- | --- |
 | {time (12 hour clock)} {AM/PM (capitalized)} | 7:58 PM CST |
 | {time (12 hour clock)} {timezone (capitalized 3 letter abbreviation)} | 12:59:16 AM CDT (UTC-5) |
 {: class="ui very basic collapsing celled table"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<hr>
 
-# Timestamp
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+#### Timestamp
+-   Add seconds or milliseconds to the time only when that level specificity is
+    important
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Timestamp format"] [class:"image bg-light border"] %}
 | Format | Examples |
 | --- | --- |
 | {month (mixed case 3 letter abbr)} {numerical day}<br>{year (4 digit)} {time (12-hr)} {AM/PM (caps)} {timezone (caps 3 letter abbr)} | Nov 28, 2016 7:58 PM CST<br>Apr 2, 2016 12:59:16 AM CDT (UTC-5) |
 {: class="ui very basic collapsing celled table"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<hr>
+
+### Timeago
 
-# Timeago
+<div class="hxRow">
+{% column left:"hxCol-8" %}
 
 Timeago details length of time that has elapsed since an event (for example, a
 monitoring alarm or ticket update). This is helpful for orienting users to the
 timeline of events.
 
-## Best Practices
+#### Best Practices
 
 -   Display order:
     -   Under 1 minute: "just now"
```

Comment:
Add a line break here.
### How it looks now
![image](https://user-images.githubusercontent.com/586682/32908533-7911c35c-cac9-11e7-9e1b-8de2ad8b1833.png)
### With line break
![image](https://user-images.githubusercontent.com/586682/32908537-7d33a630-cac9-11e7-90e9-e8f9f5932648.png)